optaplanner-examples/src/main/java/org/optaplanner/examples/cloudbalancing/persistence/CloudBalanceSolutionFileIO.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2020 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.optaplanner.examples.cloudbalancing.persistence;
+
+import org.optaplanner.examples.cloudbalancing.domain.CloudBalance;
+import org.optaplanner.persistence.xstream.impl.domain.solution.XStreamSolutionFileIO;
+
+public class CloudBalanceSolutionFileIO extends XStreamSolutionFileIO<CloudBalance> {
```

Comment:
Are these actually necessary? Maybe there are in the XML, but in the code, I was always able to get away with just using `XStreamSolutionFileIO`.

Reply:
If you use the `XStreamSolutionFileIO` directly to read a data set, sure. But the benchmark config instantiates the `SolutionFileIO` class itself.

As the constructor requires a user to pass classes that are supposed to be serialized:
```
XStreamSolutionFileIO(Class... xStreamAnnotatedClasses)
```
we would need another config code that reads these classes. And again, we are coupling the benchmark with a subset of possible `SolutionFileIO` implementations.